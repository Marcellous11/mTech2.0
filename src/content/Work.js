import React from 'react';

import '../styles/Work.css';
const Work = () => {
	return (
		<div className="Work">
			<div className="Work-Intro">
				<p className="Title">Work History</p>

				<p className="Work-Job">Software Engineering Coding Bootcamp, Remote</p>

				<p className="Work-Date">01/2022 -01/2023</p>
				<p className="Work-Position">Software Engineer</p>
				<ul className="Work-Detail">
					<li>
						<b>800+ hours</b> of hands-on intensive course material on Software Engineering methodologies
						and 1:1 industry expert mentor oversight.
					</li>
					<li>
						{' '}
						<b>Built and designed APIs</b> to feed data demanded by web server.
					</li>
					<li>
						Created web scrappers that store data into a <b>PostgreSQL database.</b>{' '}
					</li>
					<li>Practiced analytical approached to problem-solving and trouble shooting.</li>
				</ul>
				<p className="Work-Job">Personal Website “Marcellous.tech”, Remote</p>
				<p className="Work-Date">02/2023</p>
				<p className="Work-Position">Front-end developer</p>
				<ul className="Work-Detail">
					<li>
						{' '}
						Programmed website in <b>JavaScript</b> using the <b>React </b> library.
					</li>
					<li>Secured website’s SSL using Cloudflare.</li>
					<li>
						Designed website with all <b>custom CSS</b> classes.
					</li>
					<li>
						{' '}
						<b>Created web scrappers</b> that store data into PostgreSQL database.
					</li>
					<li>
						Self-motivated to create a brand that represents my{' '}
						<b style={{ color: 'white' }}>love for computers </b> and eye for design.
					</li>
				</ul>
				<p className="Work-Job">Lift Energy Solar, Fresno, CA</p>
				<p className="Work-Date">03/2021-08/2021</p>
				<p className="Work-Position"> Solar Sales Associate</p>
				<ul className="Work-Detail">
					<li> Sold 6 solar systems over the course of 3 months totaling over 40kW.</li>
					<li>Knocked on an average of 100 doors per day for 1 month.</li>
					<li>
						Streamlined sales process by performing both roles of initial contact associate and sales
						representative after 1 month.
					</li>
				</ul>

				<p className="Work-Job">Invoke Inc Public Sector, Frisco (remote), TX</p>
				<p className="Work-Date">04/2023 - Current</p>
				<p className="Work-Position">Inteligent Automation Consultant</p>
				<ul className="Work-Detail">
					<li>Goverment contracted to consult airman on builing automation. Involves remote work and onsite work at Air Force bases. Requires Common Access clearance.</li>
                    <li>Work with a variety of software including: UiPath, C#, VB, Python and more.</li>
					<li>Build and test more complex automations. Followed up with post completion support.</li>
				</ul>
			</div>
		</div>
	);
};
export default Work;
