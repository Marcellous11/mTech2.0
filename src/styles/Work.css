.Work-Job {
	color: pink;
	font: 900;
}
.Work-Title * {
	font-family: 43px;
	/* color: rgb(171, 104, 156); */
}

.Work-Job {
	color: rgb(0, 230, 0);
	font-size: 1.2em;
	font-weight: 600;
	margin-bottom: .5em;
}

.Work-Date {
	color: rgb(0, 230, 0);
	font-size: 1em;
	font-style: italic;
	margin-top: .5em;
}
.Work-Position {
	color: rgb(255, 86, 227);
	font-size: 1em;
	font-weight: 600;
}
.Work-Detail li {
	color: rgb(0, 230, 0);
	font-size: 1em;
	line-height: 1.5em;
}
@media only screen and (max-width: 830px) {
	.Work-Detail li {
		margin-left: -1.5em;
	}
}
