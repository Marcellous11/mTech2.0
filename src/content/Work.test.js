import { render, screen } from '@testing-library/react';
import Work from './Work';

test('Smoke test, does the component render', () => {
	render(<Work />);
});

test('Snapshot ', () => {
	const { asFragment } = render(<Work />);

	expect(asFragment()).toMatchSnapshot();
});

test(' "Work History" title should be on page', () => {
	const { getByText } = render(<Work />);

	expect(getByText('Work History')).toBeInTheDocument();
});

test(' "Software Engineering Coding Bootcamp, Remote" title should be on page', () => {
	const { getByText } = render(<Work />);

	expect(getByText('Software Engineering Coding Bootcamp, Remote')).toBeInTheDocument();
});
test(' "Personal Website “Marcellous.tech”, Remote" title should be on page', () => {
	const { getByText } = render(<Work />);

	expect(getByText('Personal Website “Marcellous.tech”, Remote')).toBeInTheDocument();
});
test(' "Lift Energy Solar, Fresno, CA" title should be on page', () => {
	const { getByText } = render(<Work />);

	expect(getByText('Lift Energy Solar, Fresno, CA')).toBeInTheDocument();
});
