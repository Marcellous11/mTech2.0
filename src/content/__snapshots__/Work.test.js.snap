// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot  1`] = `
<DocumentFragment>
  <div
    class="Work"
  >
    <div
      class="Work-Intro"
    >
      <p
        class="Title"
      >
        Work History
      </p>
      <p
        class="Work-Job"
      >
        Software Engineering Coding Bootcamp, Remote
      </p>
      <p
        class="Work-Date"
      >
        01/2022 -01/2023
      </p>
      <p
        class="Work-Position"
      >
        Software Engineer
      </p>
      <ul
        class="Work-Detail"
      >
        <li>
          700+ hour hands-on intensive course in Software Engineering methodologies and 1:1 industry expert mentor oversight
        </li>
        <li>
          Built and designed APIs to feed data demanded by web server.
        </li>
        <li>
          Created web scrappers that store data into PostgreSQL database.
        </li>
        <li>
          Practiced analytical approached to problem-solving and trouble shooting.
        </li>
      </ul>
      <p
        class="Work-Job"
      >
        Personal Website “Marcellous.tech”, Remote
      </p>
      <p
        class="Work-Date"
      >
        02/2023
      </p>
      <p
        class="Work-Position"
      >
        Front-end developer
      </p>
      <ul
        class="Work-Detail"
      >
        <li>
           Programmed website in JavaScript using the React library.
        </li>
        <li>
          Secured website’s SSL using Cloudflare.
        </li>
        <li>
          Designed website with all custom CSS classes.
        </li>
        <li>
          Created web scrappers that store data into PostgreSQL database.
        </li>
        <li>
          Self-motivated to create a brand that represents my love for computers and eye for design.
        </li>
      </ul>
      <p
        class="Work-Job"
      >
        Lift Energy Solar, Fresno, CA
      </p>
      <p
        class="Work-Date"
      >
        03/2021-08/2021
      </p>
      <p
        class="Work-Position"
      >
         Solar Sales Associate
      </p>
      <ul
        class="Work-Detail"
      >
        <li>
           Sold 6 solar systems over the course of 3 months totaling over 40kW.
        </li>
        <li>
          Knocked on an average of 100 doors per day for 1 month.
        </li>
        <li>
          Streamlined sales process by performing both roles of initial contact associate and sales representative after 1 month.
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
